platform/darwin/test/MGLExpressionTests.mm

```diff
@@ -300,6 +300,28 @@ - (void)testConstantValueExpressionObject {
         XCTAssertEqualObjects(expression.mgl_jsonExpressionObject, jsonExpression);
         XCTAssertEqualObjects([expression expressionValueWithObject:nil context:nil], color);
     }
+    {
+        // Transform color components to non-premultiplied values
+        float alpha = 0.5;
+        float red = (255.0 * alpha) / 255;
+        float green = (239.0 * alpha) / 255;
+        float blue = (213.0 * alpha) / 255;
+
+        MGLColor *color = [MGLColor mgl_colorWithColor:{ red, green, blue, alpha }]; // papayawhip
+        CGFloat originalRed = 0.0, originalGreen = 0.0, originalBlue = 0.0, originalAlpha = 0.0;
+        [color getRed:&originalRed green:&originalGreen blue:&originalBlue alpha:&originalAlpha];
```

Comment:
This code takes three components, creates an `mbgl::Color`, converts it to an NSColor/UIColor, then gets the components back out, calling them “originals” but doesn’t check that they match the original `float`s defined above.